Mage.Sets/src/mage/cards/l/LilianaDeathsMajesty.java

```diff
@@ -0,0 +1,96 @@
+ *  SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+ *  ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ *  NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+ *  ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ *  The views and conclusions contained in the software and documentation are those of the
+ *  authors and should not be interpreted as representing official policies, either expressed
+ *  or implied, of BetaSteward_at_googlemail.com.
+ */
+package mage.cards.l;
+
+import java.util.UUID;
+
+import mage.abilities.LoyaltyAbility;
+import mage.abilities.common.PlanswalkerEntersWithLoyalityCountersAbility;
+import mage.abilities.effects.Effect;
+import mage.abilities.effects.common.CreateTokenEffect;
+import mage.abilities.effects.common.DestroyAllEffect;
+import mage.abilities.effects.common.PutTopCardOfLibraryIntoGraveControllerEffect;
+import mage.abilities.effects.common.ReturnFromGraveyardToBattlefieldTargetEffect;
+import mage.abilities.effects.common.continuous.BecomesBlackZombieAdditionEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Duration;
+import mage.constants.Outcome;
+import mage.filter.common.FilterCreatureCard;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.filter.predicate.Predicates;
+import mage.filter.predicate.mageobject.SubtypePredicate;
+import mage.game.permanent.token.ZombieToken;
+import mage.target.common.TargetCardInYourGraveyard;
+
+/**
+ *
+ * @author JRHerlehy
+ */
+public class LilianaDeathsMajesty extends CardImpl {
+
+    private static final FilterCreaturePermanent filter = new FilterCreaturePermanent("non-Zombie creatures");
+
+    static {
+        filter.add(Predicates.not(new SubtypePredicate("Zombie")));
+    }
+
+    public LilianaDeathsMajesty(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.PLANESWALKER}, "{3}{B}{B}");
+
+        this.subtype.add("Liliana");
+
+        //Starting Loyalty: 5
+        this.addAbility(new PlanswalkerEntersWithLoyalityCountersAbility(5));
+
+        // +1: Create a 2/2 black Zombie creature token. Put the top two cards of your library into your graveyard.
+        LoyaltyAbility ability = new LoyaltyAbility(new CreateTokenEffect(new ZombieToken()), 1);
+        ability.addEffect(new PutTopCardOfLibraryIntoGraveControllerEffect(2));
+        this.addAbility(ability);
+
+        // -3: Return target creature card from your graveyard to the battlefield. That creature is a black Zombie in addition to its other colors and types.
+        ability = new LoyaltyAbility(new ReturnFromGraveyardToBattlefieldTargetEffect(), -3);
+        ability.addTarget(new TargetCardInYourGraveyard(new FilterCreatureCard("creature card from a graveyard")));
```

Comment:
Ah, too used to my Legacy reanimator.  Incoming text fix to get the text for the card to correctly generate.